.github/CODEOWNERS

```diff
@@ -1255,6 +1261,12 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 ### END Observability Plugins
 
 # Presentation
+/x-pack/test/accessibility/apps/group3/maps.ts  @elastic/kibana-presentation
+/x-pack/test/accessibility/apps/group3/graph.ts  @elastic/kibana-presentation
```

Comment:
This should be owned by @elastic/kibana-visualizations

Reply:
https://github.com/elastic/kibana/pull/195964/commits/3bfaaeda420f0a1684fc7ad88a549af1be6a35b2